templates/flower/flower-networkpolicy.yaml

```diff
@@ -23,17 +23,12 @@ spec:
       release: {{ .Release.Name }}
   policyTypes:
   - Ingress
-{{- if .Values.platform.release }}
   ingress:
   - from:
-    - namespaceSelector: {}
-      podSelector:
-        matchLabels:
-          tier: nginx
-          component: ingress-controller
-          release: {{ .Values.platform.release }}
+{{- if .Values.flower.extraNetworkPolicies }}
```

Comment:
Does this render right (and work) when extraNetworkPolicies is empty?

Reply:
I think so. I figured we could leave the ports configured regardless and optionally let users allow custom pods to communicate. So the default is that there are no sources of traffic allowed.

```# Source: airflow/templates/flower/flower-networkpolicy.yaml
################################
## Airflow Flower NetworkPolicy
#################################
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: kindly-labradoodle-flower-policy
  labels:
    tier: airflow
    component: airflow-flower-policy
    release: kindly-labradoodle
    chart: "airflow-0.12.0-alpha.5"
    heritage: Tiller
    platform: astronomer
    workspace: "123456789"
    
spec:
  podSelector:
    matchLabels:
      tier: airflow
      component: flower
      release: kindly-labradoodle
  policyTypes:
  - Ingress
  ingress:
  - from:
    ports:
    - protocol: TCP
      port: 5555
```

Reply:
Well, good point. It renders ok, but it's actually allow traffic from anyone, so I'll have to fix that.

```
$ kc describe networkpolicy intentional-lizard-flower-policy 
Name:         intentional-lizard-flower-policy
Namespace:    default
Created on:   2020-04-08 13:13:29 -0400 EDT
Labels:       chart=airflow-0.12.0-alpha.5
              component=airflow-flower-policy
              heritage=Tiller
              platform=astronomer
              release=intentional-lizard
              tier=airflow
              workspace=123456789
Annotations:  <none>
Spec:
  PodSelector:     component=flower,release=intentional-lizard,tier=airflow
  Allowing ingress traffic:
    To Port: 5555/TCP
    From: <any> (traffic not restricted by source)
  Not affecting egress traffic
  Policy Types: Ingress
```

Reply:
Well, now that I think about this again. What would we want open source default behavior to be? Should flower and the webserver allow traffic from anyone by default? Maybe that's preferable as default. :thinking:

Reply:
>Should flower and the webserver allow traffic from anyone by default? 
+1 for OSS chart.

Reply:
I think we can leave it as is for now then. By default it'll be open, and as we apply policies, it'll get locked down.